packages/tables/resources/views/components/columns/column.blade.php

```diff
@@ -16,12 +16,12 @@
     $url = $column->getUrl();
 
     $alignmentClass = match ($alignment) {
-        'center' => 'text-center',
-        'end' => 'text-end',
-        'left' => 'text-left',
-        'right' => 'text-right',
-        'justify' => 'text-justify',
-        default => 'text-start',
+        'center' => 'text-center flex justify-center',
+        'end' => 'text-end flex justify-end',
+        'left' => 'text-left flex justify-start',
+        'right' => 'text-right flex justify-end',
+        'justify' => 'text-justify flex justify-between',
+        default => 'text-start flex justify-start',
```

Comment:
Also maybe remove the text alignment classes if they dont work

Reply:
Or maybe the text alignment classes need to stay to ensure that the text column still works?

Reply:
@danharrin I just tried it out. The text alignment classes are not needed, it works with just flex and justify classes. But: alignJustify does not work either way as far as I can see, since the text itself is wrapped again in a different div that does not extend to the whole column width. Should I try to fix that as well?

Reply:
Does flex still work if the text column has `wrap()`

Reply:
> Does flex still work if the text column has `wrap()`

looks like it, yes:
![image](https://user-images.githubusercontent.com/4368880/225288407-0a1ddcd3-8bd8-4e59-8762-1bbdbb6ef3d7.png)
![image](https://user-images.githubusercontent.com/4368880/225288515-bfff63a9-6b2a-4066-8fad-5af9bac7a489.png)

Reply:
> When the text is on multiple lines though

Yeah, no. Good catch. I think it's best to leave the text alignment classes in.